search-java/src/main/java/uk/os/elements/search/SearchManager.java

```diff
@@ -190,4 +242,24 @@ public SearchBundle call(SearchBundle searchBundle, List<SearchResult> localMatc
         }
         return dest;
     }
+
+    private static AddressApi provideAddressSearchApi() {
+        Retrofit retrofit = new Retrofit.Builder()
+                // TODO
+                .baseUrl("https://api.ordnancesurvey.co.uk/places/v1/addresses/")
+                .addConverterFactory(GsonConverterFactory.create())
+                .addCallAdapterFactory(RxJavaCallAdapterFactory.create())
+                .build();
+        return retrofit.create(AddressApi.class);
+    }
+
+    private static SearchApi provideSearchApi() {
+        Retrofit retrofit = new Retrofit.Builder()
+                .baseUrl("https://api.ordnancesurvey.co.uk/opennames/v1/")
+                .addConverterFactory(GsonConverterFactory.create())
+                .addCallAdapterFactory(RxJavaCallAdapterFactory.create())
+                .build();
+        return retrofit.create(SearchApi.class);
+    }
```

Comment:
@drhaynes could talk about the above - essentially, we could bury this within the relevant provider.

The main benefit is for folk to customize this, e.g. put an interceptor on a client.  It could be the above is just a default that folk could override in the builder if they _really_ want to, e.g. to specify an ITE/beta endpoint etc.